### DISCUSSION TITLE
Does SCIP have a feature for identifying an Irreducible Inconsistent Subset (IIS) similar to Gurobi's infeasibility analysis?

### DISCUSSION THREAD

**Discussion by liuxiang2020** [scipopt/PySCIPOpt#854] [Q&A] [answered]
When I use scip, a problem is feasible, changing the constraint value of one of the constraints is considered to be an infeasible solution, but when I use gurobi to solve it, it is feasible before and after the change, so I want to see the infeasible solution analysis log of scip.

**Comment by Joao-Dionisio** [comment]
Hello, @liuxiang2020! Please see this [issue](https://github.com/scipopt/scip/issues/69). It's not similar to Gurobi's IIS, but I believe you could work with it. Adding slack variables to the constraints and minimizing their sum can tell you which constraints are leading to infeasibility. Actually, this might be something that we can add to our recipes, now that I think about it.

One explanation for the difference between the two solvers can be different numerics.

@alexhoen, is your delta debugger applicable here? 

EDIT: For anyone coming here later, I believe that the (unmerged as of now) PR #857 should aid in this sort of thing.

EDIT 2: PR #857 has now been merged.

**Comment by Joao-Dionisio** [comment, marked as answer]
The ```get_infeasibilities``` function from the recipes package provides a similar functionality. Please take a look at PR #857 for additional details.
